ddtrace/debugging/_exception/auto_instrument.py

```diff
@@ -210,3 +212,16 @@ def on_span_finish(self, span: Span) -> None:
 
             span.set_tag_str(DEBUG_INFO_TAG, "true")
             span.set_tag_str(EXCEPTION_ID_TAG, str(exc_id))
+
+
+def capture_exception() -> None:
+    import sys
+
+    from ddtrace import tracer
+
+    try:
+        span = tracer.current_span()
+        if span is not None:
```

Comment:
how this works when
1. a user call capture_exception() multiple times? will the first one/last one win? do we capture all snapshots multiple times?
2. how this works if the exception also propagate to the span it self? is that same as call "capture_exception" twice?

Reply:
Multiple calls would cause the data to be emitted multiple times. We could probably guard against this by checking if the span has `error` already set, and skip.

> how this works if the exception also propagate to the span

I'm not quite sure I understand this. With this call we are setting an error on the span. However, we don't expect this to be used when the exceptions are not handled. This will only work in an exception handler, as anywhere else in the code there won't be an exception to attach to a span. If the exception is re-raised within an exception block however, then the tracer will trigger the capturing of debug data a second time.

Reply:
You got it. Therefore, I think this logic is little broken and should emit debug info only the last/first error that was assigned to the span.